docker/tidb-sysbench/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM alpine:3.9
+MAINTAINER PingCAP <cloud@pingcap.com>
+
+# Install sysbench
+# gcc and mariadb-dev install shared libraries used at runtime
+RUN apk add gcc mariadb-dev
+
+RUN apk add --virtual .build-deps git build-base automake autoconf libtool --update \
+  && git clone https://github.com/akopytov/sysbench.git \
+  && cd sysbench \
+  && git checkout 1.0.17 \
```

Comment:
Make the version as an environment variable?

Reply:
okay